### ISSUE TITLE
If conversion fails, the app freezes up

### ISSUE THREAD

**Issue by ChildishGiant** [squeaksies/MediocreMapper#71]
https://gfycat.com/tatteredweepyfulmar
![tatteredweepyfulmar](https://thumbs.gfycat.com/TatteredWeepyFulmar-size_restricted.gif)

Here's the map that fails. [Prom Dress.zip](https://github.com/squeaksies/MediocreMapper/files/3580296/Prom.Dress.zip) My guess is that because it has no maps, only an info.json, the converter gets in a tizz and keeps trying.

Ideally, the user gets told that conversion failed (maybe even with a reason) and sent back to the song list.

**Comment by squeaksies** [comment]
Yeah, i literally forgot to make the button do anything. Will be resolved soon
